app/views/course/leaderboards/show.html.slim

```diff
@@ -1 +1,43 @@
-p To be implemented later
+= page_header t('course.leaderboard.index.header')
```

Comment:
I prefer the convention, otherwise the controller should be a singular controller.